core/templates/components/certificate-assessment-offering-helper/certificate-offering-review-and-availability.component.css

```
@@ -0,0 +1,206 @@
+.oppia-certificate-review-container {
+  color: #333;
+  font-family: "Roboto", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)